packages/flutter/lib/src/gestures/hit_test.dart

```diff
@@ -2,8 +2,15 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:collection';
+
+import 'package:flutter/foundation.dart';
+import 'package:vector_math/vector_math_64.dart';
+
 import 'events.dart';
 
+typedef HitTest = bool Function(HitTestResult result, Offset position);
```

Comment:
this is only RenderBox's hitTest type... I don't think the gestures library should hard-code this.